packages/footer/components/LinkData.tsx

```diff
@@ -0,0 +1,191 @@
+import { FooterType } from './Shared';
```

Comment:
I'm not 100% sure, but if these are types, we should be doing `import type { FooterType } ...`

This applies to all imports. I don't think it matters functionally, but it may be what's inflating the bundle size on this branch?

Reply:
[893cac9](https://github.com/iFixit/react-commerce/pull/1342/commits/893cac98ed4ef1328248455754490c41ab0108f7)